tests/braille-specs/en-ueb.yaml

```diff
@@ -1288,6 +1285,68 @@ tests:
       italic: '                   ++++++++++++++++++++++++++++++++ ++++++++++++++++++++++'
       bold:   '                   ++++++++++++++++++++++++++++++++                       '
 
+
+
+# from issue #1465: positioning of closing typeform indicators
+- - "Answer all questions."
+  - ~7,answ} all "qs4~'
+  - typeform:
+      bold: '++++++++++++++++++++ '
+
+    xfail: true
+- - "Answer all questions."
+  - '~7,answ} all "qs4~'''
+  - typeform: {bold: '+++++++++++++++++++++'}
```

Comment:
@jrbowden You said:

> Note: please fix the test at lines 1295-99
> I have no idea why I had to misplace "xfail:true" in order to make this work.
> All the other xfail lines come after typeform, but if I do that, the system
> seems in error to want to apply xfail to the following test.

There is nothing wrong with the system. The `xfail` is applied to the preceding test. In this case, the first test fails and the second one passes. Can you double check the expected braille, and can you tell me what the expected braille is without any quoting and quote escaping that YAML requires? Just the plain ASCII braille.